### PR TITLE
Comment

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
added comment feat
added edit profile feat
added trending/latest feat

### CODE REVIEW

File: src/features/auth/AuthSlice.js [hunk 1]
```diff
@@ -158,16 +158,16 @@ const authSlice = createSlice({
...
-    // },
-    // [updateUser.fulfilled]: (state, action) => {
-    //   state.user = action.payload;
-    //   localStorage.setItem("user", JSON.stringify(state.user));
-    // },
-    // [updateUser.rejected]: (state, action) => {
-    //   state.error = action.payload;
-    // },
+    [updateUser.pending]: (state) => {
+      console.log(state);
```
Comment (Pratik1005) [inline src/features/auth/AuthSlice.js:162]: remove log if not needed

File: src/features/post/PostSlice.js [hunk 2]
```diff
@@ -145,6 +145,73 @@ export const removePostFromBookmark = createAsyncThunk(
...
+      return response.data;
+    } catch (error) {
+      return thunkAPI.rejectWithValue(error)
+    }
+  }
+);
+export const editComment = createAsyncThunk(
+  "post/editComment",
+  async ({ postId,commentId,commentData }, thunkAPI) => {
+    console.log("Click")
```
Comment (Pratik1005) [inline src/features/post/PostSlice.js]: same here

File: src/features/post/PostSlice.js [hunk 3]
```diff
@@ -145,6 +145,73 @@ export const removePostFromBookmark = createAsyncThunk(
...
+        {
+          commentData
+        },
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      console.log(response.data)
```
Comment (Pratik1005) [inline src/features/post/PostSlice.js]: same here

**Review by Pratik1005 — APPROVED** [review]
When we comment first time and edit it, the edit text is different, edit comment can take empty value, rest looks good

**Review by Mandhularajitha — COMMENTED** [review]
Code looks clean,good to go

**Review by Mandhularajitha — COMMENTED** [review]
Code looks clean,good to go

**Pull request merged**